doc/help.md

```diff
@@ -889,6 +889,86 @@ JOB:
 
 ### External
 
+<h4 id="[">[</h4>
+
+    [ EXPR+ ]
+
+This is an alias for the `test` builtin. Unlike `test`, expressions must be
+followed by a closing ']'.
+
+Tips:
+
+Don't forget to leave a space between the brackets and the expression. "[" is
+a command, not syntax, so the shell will fail to recognize it without the
+spaces.
+
+
+<h4 id="test">test</h4>
+
+    test EXPR+
```

Comment:
What's EXPR?  It can't be repeated one or more times (`+`).

I think it's more accurate to write as:

```
test STR
test OP STR
test STR OP STR

[ STR ]
[ OP STR ]
[ STR OP STR ]
```

Reply:
That seems sensible, I added that and also equivalents for other tests, which I guess they are necessary?